src/sdk/pynni/nni/compression/torch/builtin_pruners.py

```diff
@@ -319,31 +334,32 @@ def _get_mask(self, base_mask, weight, num_prune):
         Filters with the smallest sum of its absolute kernel weights are masked.
         Parameters
         ----------
-        base_mask : torch.Tensor
-            The basic mask with the same shape of weight, all item in the basic mask is 1.
+        base_mask : dict
+            The basic mask with the same shape of weight or bias, all item in the basic mask is 1.
         weight : torch.Tensor
             Layer's weight
         num_prune : int
             Num of filters to prune
         Returns
```

Comment:
add a new blank line between line 342 and line 343

Reply:
resolved